### ISSUE TITLE
AttributeError: 'module' object has no attribute 'QStringListModel'

### ISSUE THREAD

**Issue by Shayen** [jchanvfx/NodeGraphQt#92]
When I try to run [example.py](https://github.com/jchanvfx/NodeGraphQt/blob/master/example.py) in Autodesk Maya 2017 (And still happen in Maya 2018 also), It throw this error:
```"D:\Program Files\Autodesk\Maya2017\bin\mayapy.exe" D:/WORK/Programming/graphapp/NodeGraphQt/example.py
Traceback (most recent call last):
  File "D:/WORK/Programming/graphapp/NodeGraphQt/example.py", line 37, in <module>
    graph = NodeGraph()
  File "D:\WORK\Programming\graphapp\NodeGraphQt\NodeGraphQt\base\graph.py", line 42, in __init__
    self._viewer = NodeViewer()
  File "D:\WORK\Programming\graphapp\NodeGraphQt\NodeGraphQt\widgets\viewer.py", line 65, in __init__
    self._search_widget = TabSearchWidget(self)
  File "D:\WORK\Programming\graphapp\NodeGraphQt\NodeGraphQt\widgets\tab_search.py", line 63, in __init__
    self._model = QtCore.QStringListModel(node_names, self)
AttributeError: 'module' object has no attribute 'QStringListModel'
```
After googling for a while I found [the solution](https://github.com/pytest-dev/pytest-qt/issues/205#issuecomment-381223150) from another repo.

In this case I just change
`self._model = QtCore.QStringListModel(node_names, self)`
To
`self._model = QtGui.QStringListModel(node_names, self)`

then its work for me.

By the way, When I run [example.py](https://github.com/jchanvfx/NodeGraphQt/blob/master/example.py) with normal python3.7 and pyside2(5.12.0) outside Maya, It's work properly with the original code.

**Comment by jchanvfx** [comment]
hi @Shayen 

Here's a thread from another project that might explain.
https://github.com/mottosso/Qt.py/pull/302

**Comment by Shayen** [comment]
Hi @jchanvfx,

Did you have any plan to support the old version of PySide2?
like Maya 2017 and 2018, It's using version 2.0.0~alpha0 and 2.0. But the bug has been fixed on 5.6 and 5.9.

**Comment by jchanvfx** [comment]
Hi @Shayen 

Sadly no I don't plan to do any back porting as I'm only working on this project in the spare time I have.

J
